composer.lock

```diff
@@ -7286,8 +7286,8 @@
                     "dev-3.x": "3.x-dev"
                 },
                 "drupal": {
-                    "version": "8.x-3.0-beta8+12-dev",
-                    "datestamp": "1505482144",
+                    "version": "8.x-3.0-beta8+31-dev",
```

Comment:
I cant tell whats going on here. Did purge module update as well?

Reply:
It looks like it. I think we might want to try this one again.`

Reply:
I did a "targeted" updated:
```
$ composer require symfony/http-foundation:3.4.35 --no-update
$ composer symfony/http-foundation  # <<<<<<==== no update with dependencies here
```
It does look like `composer` still got us to a new beta version of `drupal/purge`. It might give us more "stability", and unless you think otherwise, I'll drop the `drupal/purge` beta bump from this PR.

Reply:
Done. Cleanup has been performed.